src/App.jsx

```diff
@@ -1,7 +1,22 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom"
+import HomePage from "./Pages/HomePage"
+import Header from "./Components/Header"
+import NewEmail from "./Pages/NewEmail/Index"
+
 function App () {
     return (
         <div>
-            <h1>asasa</h1>
+            
+           <BrowserRouter>
+                <Header />
+                <Routes>
+                    <Route path="/" element={<HomePage/>} />
+                    <Route path="/email" element={<NewEmail />} />
```

Comment:
Can you remove this route please - we don't want anything unfinished getting into main